mlflow/models/evaluation/default_evaluator.py

```diff
@@ -612,6 +613,7 @@ class _Metric(NamedTuple):
     name: str
     index: int
     version: Optional[str] = None
+    genai_metric_args: Optional[dict] = None
```

Comment:
The type hint for a dictionary in Python is `Dict` from the typing library. `dict` is a reserved function name. 
(weird language, I know ;) )

Reply:
😵‍💫